### ISSUE TITLE
Manque SDIS et métropole du Grand Nancy

### ISSUE THREAD

**Issue by samgoeta** [okfnfr/dada-france-authorities#6]
https://teamopendata.org/t/lancement-de-madada-fr-un-portail-pour-faciliter-les-demandes-de-donnees/1715/5?u=samgoeta

**Comment by akakeronos** [comment]
à ce sujet j'aimerai savoir s'il y a un moyen d'importer plusieurs autorités d'un coup ?

**Comment by laurentS** [comment]
@akakeronos oui, tu les mets dans un fichier CSV, puis https://madada.fr/admin/bodies/import_csv
Le parser du fichier est plutôt tatillon, je te conseille de faire des dry-run jusqu'à ce que tu n'aies plus d'erreurs. Le format du CSV est expliqué sur cette page.

**Comment by akakeronos** [comment]
ok merci !

**Comment by pzwsk** [comment]
Manque les emails

**Comment by samgoeta** [comment]
Il me manque aussi un interlocuteur à la métropole Aix Marseille : https://madada.fr/demande/donnees_des_capteurs_de_comptage#incoming-3
